markdown/ch7.markdown

```diff
@@ -649,7 +682,7 @@ for ax in (ax0, ax1):
 What a glorious image! Realise that this artifact was created before color
 photography existed! Notice God's pearly white robes, John's white beard,
 and the white pages of the book held by Prochorus, his scribe — all of which
-were missing from the MSE-based alignment.
+were missing from the MSE-based alignment, but nevertheless look clear.
```

Comment:
It was unclear, why is it relevant that they were missing from the MSE-based alignment? I assumed your point was that once you lined up the colours, the white falls into place. So they white is clear despite the fact that you didn't explicitly work on them.